etabotsite/etabotapp/views.py

```diff
@@ -38,23 +38,48 @@
 else:
     logger.setLevel(logging.INFO)
 
+PROD_HOST_URL = getattr(settings, "PROD_HOST_URL", "http://localhost:8000")
+
+
 def fetch_token(name, request):
+    """Authlib support function."""
+    OAUTH1_SERVICES = []
     if name in OAUTH1_SERVICES:
         model = OAuth1Token
     else:
         model = OAuth2Token
```

Comment:
The logic here seems to be redundant to me. `OAUTH1_SERVICES` is an empty list, the first condition always holds, `model` will always be `OAuth1Token`. Please revisit the logic here.

Reply:
it is a recommended pattern from developer. OAUTH1_SERVICES will be populated in future if needed